parts/JavaConfig-JSP/projectName-web/src/main/java/xxxxxx/yyyyyy/zzzzzz/config/web/SpringSecurityConfig.java

```diff
@@ -41,13 +41,16 @@ public WebSecurityCustomizer webSecurityCustomizer() {
      */
     @Bean
     public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
-        http.formLogin();
-        http.logout();
+        http.formLogin(login -> {
+        });
+        http.logout(logout -> {
+        });
         http.exceptionHandling(ex -> ex
                 .accessDeniedHandler(accessDeniedHandler()));
         http.addFilterAfter(
                 userIdMDCPutFilter(), AnonymousAuthenticationFilter.class);
-        http.sessionManagement();
+        http.sessionManagement(sessionManagement -> {
+        });
 
         return http.build();
     }
```

Comment:
- [ ] In accordance with the way Spring Security is written, it could be written as follows.
```
import static org.springframework.security.config.Customizer.withDefaults;

// omitted

    @Bean
    public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
        http.formLogin(login -> withDefaults());
        http.logout(logout -> withDefaults());
        http.exceptionHandling(ex -> ex.accessDeniedHandler(
                accessDeniedHandler()));
        http.addFilterAfter(userIdMDCPutFilter(),
                AnonymousAuthenticationFilter.class);
        http.sessionManagement(sessionManagement -> withDefaults());

        return http.build();
    }
```

Reply:
69407af96094a8b050ca3f53d64d99908e8664cc
use `Customizer#withDefaults()`